inspec/aws/aws_list.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+aws-list-all query --region us-east-2 
```

Comment:
maybe passing a region as argument would be a better approach than hardcoding one